netci.groovy

```diff
@@ -125,7 +125,7 @@ branchList.each { branchName ->
                     // Jobs run as a service in unix, which means that HOME variable is not set, and it is required for restoring packages
                     // so we set it first, and then call build.sh
                     steps {
-                        shell("HOME=\$WORKSPACE/tempHome ./build.sh /p:ConfigurationGroup=${configurationGroup} /p:WithCategories=\"InnerLoop;OuterLoop\" /p:TestWithLocalLibraries=true")
+                        shell("HOME=\$WORKSPACE/tempHome ./build.sh /p:ConfigurationGroup=${configurationGroup} /p:WithCategories=\"\\\"InnerLoop;OuterLoop\\\"\" /p:TestWithLocalLibraries=true")
```

Comment:
Because of a bug in .Net Core MSBuild where it doesn't recognize the quotes so it assumes that everything after the colon is a new switch. You can see more details in Microsoft/MSBuild#471. This is basically a workaround for now, which will be removed once that MSBuild issue is fixed